modules/server/src/main/scala/higherkindness/compendium/db/FileDBService.scala

```diff
@@ -23,17 +23,16 @@ import higherkindness.compendium.models.Protocol
 import higherkindness.compendium.models.ProtocolAlreadyExists
 import higherkindness.compendium.storage.Storage
 
-object DBServiceStorage {
+object FileDBService {
 
   def impl[F[_]: Sync: Storage]: DBService[F] =
     new DBService[F] {
-
       override def addProtocol(id: String, protocol: Protocol): F[Unit] =
         for {
           exists <- Storage[F].checkIfExists(id)
           _ <- exists.fold(
             Sync[F].raiseError(new ProtocolAlreadyExists(s"Protocol with id ${id} already exists")),
-            Storage[F].store(id, protocol))
+            Sync[F].unit)
```

Comment:
we're not storing the file at this point anymore?

Reply:
Because `FileDBService` is suppose to be the one persisting the meta information to then retrieve the protocols from the `Storage`.

`FileStorage` is the one that is really saving the protocol in the file system: https://github.com/higherkindness/compendium/blob/master/modules/server/src/main/scala/higherkindness/compendium/core/CompendiumService.scala#L40